# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ipvsadm2/linux-header-install-ip_vs.h.diff
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Author: Christian Wiese <morfoh@opensde.org>
Origin: OpenSDE (http://opensde.org)
Description: Install a 'sanitized' version of the ip_vs.h header file
             to satisfy ipvsadm2 and keepalived.

diff -ruN linux-2.6.22.orig/include/Kbuild linux-2.6.22/include/Kbuild
--- linux-2.6.22.orig/include/Kbuild	2007-07-09 02:32:17.000000000 +0300
+++ linux-2.6.22/include/Kbuild	2008-03-13 20:33:16.000000000 +0200
@@ -3,6 +3,7 @@
 header-y += scsi/
 header-y += sound/
 header-y += mtd/
+header-y += net/
 header-y += rdma/
 header-y += video/
 
diff -ruN linux-2.6.22.orig/include/net/Kbuild linux-2.6.22/include/net/Kbuild
--- linux-2.6.22.orig/include/net/Kbuild	1970-01-01 02:00:00.000000000 +0200
+++ linux-2.6.22/include/net/Kbuild	2008-03-13 20:35:34.000000000 +0200
@@ -0,0 +1 @@
+unifdef-y += ip_vs.h
